lib/suse/connect/zypper.rb

```diff
@@ -84,6 +84,8 @@ def find_products(identifier)
           zypper_out = call("--xmlout --non-interactive search -s -t product #{Shellwords.escape(identifier)}", false)
           xml_doc = REXML::Document.new(zypper_out, compress_whitespace: [])
           xml_doc.elements.to_a('stream/search-result/solvable-list/solvable').map(&:to_hash)
+        rescue
```

Comment:
maybe we can do here `rescue Connect::ZypperError` to be able to be more specific